chapters/06-summary.Rmd

```diff
@@ -0,0 +1,25 @@
+# Summary {#summary}
+
+The benchmarking has shown that the new implementation can outperform the de-facto state of the art library 'KDEpy' in terms of runtime and accuracy for large datasets ($n \geq 10^8$).
```

Comment:
[9] Remove and distribute this sentence. It summarizes everything to shortly. But the summary should be a whole summary. Start with what was implemented, what was tested and the results and then the outlook